app/core/Router.coffee

```diff
@@ -122,6 +122,7 @@ module.exports = class CocoRouter extends Backbone.Router
 
     'teachers': go('TeachersView')
     'teachers/freetrial': go('TeachersFreeTrialView')
```

Comment:
Old /freetrial link should go to new quote page as well.  That link might have been shared directly.

We can also delete the old view now, or punt that to a later commit.